webviz_subsurface/plugins/_inplace_volumes.py

```diff
@@ -59,14 +59,68 @@ class InplaceVolumes(WebvizPluginABC):
 """
 
     TABLE_STATISTICS = [
-        "response",
-        "group",
-        "mean",
-        "stddev",
-        "minimum",
-        "p90",
-        "p10",
-        "maximum",
+        ("Response", {}),
+        ("Group", {}),
+        (
+            "Mean",
+            {
+                "type": "numeric",
+                "format": {
+                    "locale": {"symbol": ["", "unit_insert"]},
+                    "specifier": "$.4s",
+                },
+            },
+        ),
+        (
+            "Stddev",
+            {
+                "type": "numeric",
+                "format": {
+                    "locale": {"symbol": ["", "unit_insert"]},
+                    "specifier": "$.4s",
+                },
+            },
+        ),
+        (
+            "Minimum",
+            {
+                "type": "numeric",
+                "format": {
+                    "locale": {"symbol": ["", "unit_insert"]},
+                    "specifier": "$.4s",
+                },
+            },
+        ),
+        (
+            "P90",
+            {
+                "type": "numeric",
+                "format": {
+                    "locale": {"symbol": ["", "unit_insert"]},
+                    "specifier": "$.4s",
+                },
+            },
+        ),
+        (
+            "P10",
+            {
+                "type": "numeric",
+                "format": {
+                    "locale": {"symbol": ["", "unit_insert"]},
+                    "specifier": "$.4s",
+                },
+            },
+        ),
+        (
+            "Maximum",
+            {
+                "type": "numeric",
+                "format": {
+                    "locale": {"symbol": ["", "unit_insert"]},
+                    "specifier": "$.4s",
+                },
+            },
+        ),
```

Comment:
Use list comprehension here? As the statistics use the same dictionary.

Reply:
See next comment